src/sc/embedder/content/embedder.py

```diff
@@ -77,6 +78,28 @@ def EmbedderImageFieldWidget(field, request):
     return FieldWidget(field, EmbedderImageWidget(request))
 
 
+def valid_size_constraint(size):
+    """Validate if size inserted is a float number or a %.
+
+    :param size: size to be validated
+    :type size: unicode
+    :return: True if size is valid
+    :rtype: bool
+    :raises:
+        :class:`~zope.interface.Invalid` if the size is not valid
+    """
+
+    if size:
+        if ' ' in size:
```

Comment:
A constraint don't work like this.. I'll need to override the confirm button to change the content of a variable.

How about to change the content of the variable with javascript (take out spaces) and keep the validation?